app/models/service.rb

```diff
@@ -65,6 +65,7 @@ class Service < ApplicationRecord
   include TenancyMixin
   include SupportsFeatureMixin
   include Metric::CiMixin
+  include CustomActionsMixin
```

Comment:
Services get their custom buttons through the `service_template` ([app/models/service.rb#L56](https://github.com/ManageIQ/manageiq/blob/master/app/models/service.rb#L56)) model which already includes this mixin.  This is not needed.